websocket_server/Config.js

```diff
@@ -51,6 +51,8 @@ const Config = {
 
 	MAX_BACKUPS_PER_ROOM: process.env.MAX_BACKUPS_PER_ROOM || DEFAULT_MAX_BACKUPS_PER_ROOM,
 
+	MAX_UPLOAD_FILE_SIZE: process.env.MAX_UPLOAD_FILE_SIZE * (10 ** 6) || 2e6,
```

Comment:
```suggestion
	MAX_UPLOAD_FILE_SIZE: process.env.MAX_UPLOAD_FILE_SIZE * (1e6) || 2e6,
```

Any specific reason for the two different notations used?